### PR TITLE
Update create-your-first-service.md

### PR DESCRIPTION
**Pull request by RukshiW**
Update guide to reflect the latest UI changes.

### CODE REVIEW

File: en/docs/services/create-your-first-service.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# Create Your First Service
+# Create Your First Choreo Service
```
Comment (anupama-pathirage) [inline en/docs/services/create-your-first-service.md:1]: We don't need to say choreo explicitly

File: en/docs/services/create-your-first-service.md [hunk 2]
```diff
@@ -18,70 +18,75 @@ To create the Choreo service, follow the procedure below:
 
 4. In the **Configure API Trigger** form, select **GET** as the HTTP method, and enter `activecases` as the relative path from host.
 
-    ![Resource Configuration](../assets/img/services/configure-api-trigger.png)
+    ![Resource Configuration](/en/assets/img/services/configure-api-trigger.png)
```
Comment (anupama-pathirage) [inline en/docs/services/create-your-first-service.md:21]: Use relative paths. Check all the places. Changes were added in https://github.com/wso2/docs-choreo-dev/pull/63/files

**Comment by RukshiW** [comment]
Resolves https://github.com/wso2-enterprise/choreo/issues/3898

**Comment by RukshiW** [comment]
Corrects https://github.com/wso2-enterprise/choreo/issues/3845

File: en/docs/services/create-your-first-service.md (hunk 1 above)
Comment (RukshiW) [reply en/docs/services/create-your-first-service.md:1]: Corrected in #71

**Comment by anupama-pathirage** [comment]
Can we merge the two PRs. This one and https://github.com/wso2/docs-choreo-dev/pull/71 as it updates the same files

**Comment by anupama-pathirage** [comment]
@RukshiW It seems this PR contains lot of invalid changes (invalid topics, invalid image paths etc) Also most of the changes in this PR are also there in https://github.com/wso2/docs-choreo-dev/pull/71/ as well. So close this PR and whatever the additional valid things in this PR, move them to #71

**Pull request closed without merging**